### PR TITLE
schematic documentation

### PR DESCRIPTION
**Pull request by jasdeepg**
### Proposed changes

Added Schematic docs based on template.

### Merge timing
ASAP once approved

### CODE REVIEW

File: src/connections/destinations/catalog/schematic/index.md
Comment (forstisabella) [inline src/connections/destinations/catalog/schematic/index.md:1]: hi @jasdeepg - please update the file folder to match the slug that this integration has in the app: `actions-schematic`.

Comment (jasdeepg) [reply src/connections/destinations/catalog/schematic/index.md:1]: just did both - thanks!

File: src/connections/destinations/catalog/actions-schematic/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Schematic (Actions) Destination
+---
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-schematic/index.md:3]: hi @jasdeepg - the beta/hidden attributes denote that this integration is in private beta and the ID autogenerates some of the actions content when this integration enters public beta. 
```suggestion
---
title: Schematic (Actions) Destination
hidden: true
beta: true
id: 65b8e9eca1b5903a031c6378
---
```

File: src/connections/destinations/catalog/actions-schematic/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[Schematic](https://schematichq.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} enables you to launch, package, meter, and monitor features with ease, so you can manage it all in one place as your business grows.
+
+Segment is the easiest way to send events from your application to Schematic. If you already have Segment up and running in your applicaiton, Schematic recommends this approach so you don't have to implement any additional code.
+
+Schematic maintains this destination. For any issues with the destination, [contact the Schematic Support team](mailto:hi@schematichq.com).
+
+## Getting started
+
+1. From your Segment web app, navigate to **Connections > Catalog > Destinations**.
+2. Search for *Schematic* and select **Add Destination**
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-schematic/index.md:16]:
```suggestion
2. Search for *Schematic*, select the Schematic destination, and click **Add destination**.
```

File: src/connections/destinations/catalog/actions-schematic/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+3. Select the source that will send data to Schematic and follow the steps to name your destination. The web source chosen must use [Analytics.js 2.0](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/)
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-schematic/index.md:17]: As of March 2023, [Segment upgraded all sources to Analytics.js 2.0](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/upgrade-to-ajs2/), so I'll remove that sentence (and update the template!)

```suggestion
3. Select the source that will send data to Schematic, give your destination a name, then click **Create destination**.
```

File: src/connections/destinations/catalog/actions-schematic/index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+4. On the Settings tab, input your API Key which can be found in your Schematic workspace settings under **Settings > API Keys**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-schematic/index.md:18]:
```suggestion
4. On the destination's Settings tab, input your Schematic API Key. To generate an API key, navigate to your Schematic workspace settings under **Settings > API Keys**.
```

File: src/connections/destinations/catalog/actions-schematic/index.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+5. Once connected, you can configure how you want to send data to Schematic in the **Mappings** tab.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-schematic/index.md:19]:
```suggestion

Once you've connected Schematic to Segment, you can configure how you want to send data to Schematic in the Schematic destination's **Mappings** tab.
```

File: src/connections/destinations/catalog/actions-schematic/index.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+{% include components/actions-fields.html %}
+
+## Additional Context
+
+For track events, Schematic only accepts event names that contain alphanumeric characters, dashes, and underscores. If Segment event names have, for instance, spaces, the Schematic destination will automatically snake case the event name before passing to Schematic. Additionally, the raw event name will always be passed as an event trait.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-schematic/index.md:25]:
```suggestion
Schematic only accepts Track event names that contain alphanumeric characters, dashes, and underscores. If Segment event names have other characters, like spaces, the Schematic destination automatically snake_cases the event name before passing to Schematic. Segment passes the raw event name as an event trait.
```

**Review by forstisabella — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/actions-schematic/index.md (hunk 1 above)
Comment (jasdeepg) [reply src/connections/destinations/catalog/actions-schematic/index.md:3]: hi @forstisabella ! 1 customer has tested this integration successfully. would be great if we could have this in public beta / GA if possible.

Comment (tcgilbert) [reply src/connections/destinations/catalog/actions-schematic/index.md:3]: @jasdeepg I see you have submitted all the needed materials to go live as well. With that in mind, I removed the hidden property. You will still need to add th beta and id props

**Review by tcgilbert — CHANGES_REQUESTED** [review]

**Comment by jasdeepg** [comment]
hi - i believe all changes are in, but let me know if i missed anything.

**Review by tcgilbert — APPROVED** [review]

**Review by forstisabella — APPROVED** [review]

**Comment by tcgilbert** [comment]
@jasdeepg 
 
I have gone ahead and made your integration live. Twilio Segment is excited to move your new integration into the Public Catalog! 
 
To move out of beta, you can write back into partner-support@segment.com once you have at least one customer using the integration, and you are confident that the beta status is no longer applicable.
 
If you have any questions or concerns, please let me know.

**Pull request merged**